bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -17,7 +17,7 @@ metadata:
   {{- end }}
 spec:
   replicas: {{ .Values.postgresql.replicaCount }}
-  podManagementPolicy: Parallel
+  podManagementPolicy: {{ .Values.postgresql.podManagementPolicy | default "Parallel" }}
```

If the default value in `values.yaml` is `Parallel`, I think that we can safely do the following:

```suggestion
  podManagementPolicy: {{ .Values.postgresql.podManagementPolicy | quote }}
```